config/co2eq_parameters.json

```
@@ -179,9 +179,14 @@
         }
       },
       "CA-AB": {
-        "hydro discharge": {
-          "source": "2017 average by Tomorrow",
-          "value": 597.2654373454017
+        "unknown": {
+          "_comment": "dual fuel power plants: assumption of 50% coal and 50% natural gas",
+          "value": 655
+        }
+        ,
+        "battery discharge": {
+          "source": "https://www.cer-rec.gc.ca/en/data-analysis/energy-markets/provincial-territorial-energy-profiles/provincial-territorial-energy-profiles-alberta.html",
```

Isn't the source for the 597 figure still the "2017 average by Tomorrow"?

---

Yes it is. Is there any chance that you could calculate the 2020 average? Since 2017 a few coal power plants switched to gas or were shut down and I guess it‘s lower now. Probably more like 500.

---

> Yes it is. Is there any chance that you could calculate the 2020 average? Since 2017 a few coal power plants switched to gas or were shut down and I guess it‘s lower now. Probably more like 500.
@skovhus should we create an internal issue on our side?

Aside from that, I suggest we change the source so it properly reflects the origin of the 597 figure:
```suggestion
          "source": "2017 average by Tomorrow",
```

If we do any updates, we will probably make a new PR